scripts/docker/Dockerfile

```diff
@@ -4,7 +4,12 @@ MAINTAINER Devis Lucato (https://github.com/dluc)
 
 LABEL Tags="Azure,IoT,Solutions,Simulation,.NET"
 
+ARG user=pcsuser
```

Comment:
I think we changed this to another name, right?

Reply:
I didn't check all the repos but few and we are using pcsuser, i think it was something else before.

Reply:
we use pcsuser initially before using app.  Maybe we can change it to be consistent.